### PR TITLE
title details

### PR DESCRIPTION
**Pull request by NadiaArroyo25**

### CODE REVIEW

File: app/views/appointments/edit.html.erb [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
+  <div class="title-container">
+    <h1>Editar cita</h1>
+  </div>
 
 <%= render "form", appointment: @appointment %>
-
-  <%= link_to "Muestra esta cita", @appointment %> |
-  <%= link_to "Regresa a las citas ", appointments_path %>
+<%= link_to "Muestra esta cita", @appointment %> |
+<%= link_to "Regresa a las citas ", appointments_path %>
```
Comment (Ligator) [inline app/views/appointments/edit.html.erb]: Indentation.

File: app/views/appointments/new.html.erb
Comment (Ligator) [inline app/views/appointments/new.html.erb:1]: Indentation.

File: app/views/patients/show.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="container">
-  <p style="color: green"><%= notice %></p>
+  <p class="text-success"><%= notice %></p>
```
Comment (Ligator) [inline app/views/patients/show.html.erb:2]: 👏

**Review by Ligator — APPROVED** [review]

File: app/views/appointments/new.html.erb
Comment (NadiaArroyo25) [reply app/views/appointments/new.html.erb:1]: I think I got confused

**Pull request merged**